javascript/pane.js

```diff
@@ -81,6 +80,10 @@ function Pane(options){
     view.hide()
     $(context).trigger('hidden')
   }
+  function toggle(){
+    if (isOpen) context.hide()
+    else        context.show()
```

Comment:
Purely a style consistency change. 
```suggestion
    if (isOpen) {
      context.hide()
    } else {
      context.show()
    }
```

Also, could you move this above the functions it calls so that every call goes "down" the file?